src/Pages/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+const Homepage = () => {
+  return (
+    <div>
+      <img
+        className="cover"
+        src={require("../../Assets/cover.jpg")}
```

Comment:
I would suggest not using `require()` here. Instead, you can directly pass the Image src like this:
```
src="../../Assets/cover.jpg"
```